kernelci/kbuild.py

```diff
@@ -556,6 +654,24 @@ def submit(self, retcode):
         artifacts = []
         for artifact in self._artifacts:
             artifacts.append(os.path.join(self._af_dir, artifact))
+        # Add full artifacts path to metadata.json
+        # We do it after full artifacts upload, twice
+        # as we can get urls of artifacts AFTER upload
+        self._update_metadata(job_result)
+        # Upload metadata.json to storage
+        metadata_uri = self.upload_metadata()
+        af_uri['metadata'] = metadata_uri
+
+        # do we have dtbs/ in af_uri?
+        dtb_present = False
```

Comment:
Ah, one thing I forgot to mention: this `dtb_present` is actually unneeded, as we can just check if `platform_config` has a `dtb` key. It won't hurt to keep it, but this parameter isn't used.